src/status/chat.nim

```diff
@@ -127,6 +129,30 @@ proc updateContacts*(self: ChatModel, contacts: seq[Profile]) =
 
 proc init*(self: ChatModel, pubKey: string) =
   var chatList = status_chat.loadChats()
+  var contacts = getAddedContacts()
+
+  let profileUpdatesChatIds = chatList.filter(c => c.chatType == ChatType.Profile).map(c => c.id)
+
+  if chatList.filter(c => c.chatType == ChatType.Timeline).len == 0:
+    var timelineChannel = newChat(status_utils.getTimelineChatId(), ChatType.Timeline)
+    self.join(timelineChannel.id, timelineChannel.chatType)
+    chatList.add(timelineChannel)
+
+  if not profileUpdatesChatIds.contains("@" & pubKey):
```

Comment:
This concatenation is used in different places in the code. What do you think of adding it into utils.nim?
`proc timelineChatId*(pubKey:string): "@" & pubKey`.  It makes it easier to see what's the purpose of the concatenation that way